srcpkgs/cantata/template

```diff
@@ -1,20 +1,26 @@
 # Template file for 'cantata'
 pkgname=cantata
-version=2.5.0
-revision=2
+version=3.2.0
+revision=1
 build_style=cmake
-hostmakedepends="pkg-config qt5-qmake qt5-host-tools qt5-tools"
-makedepends="qt5-devel qt5-tools-devel qt5-svg-devel qt5-xmlpatterns-devel
- qt5-plugin-mysql qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-odbc
- qt5-plugin-tds vlc-devel zlib-devel ffmpeg6-devel taglib-devel
- taglib-extras-devel libmtp-devel mpg123-devel libcdparanoia-devel libcddb-devel
- speex-devel eudev-libudev-devel udisks2-devel libmusicbrainz5-devel
- qt5-multimedia-devel libcdio-paranoia-devel"
-depends="media-player-info qt5-plugin-sqlite"
+configure_args="-DENABLE_HTTP_STREAM_PLAYBACK=ON
+ -DENABLE_LIBVLC=OFF
+ -DENABLE_SIMPLE_MPD_SUPPORT=ON
+ -DCMAKE_BUILD_TYPE=None
+ -DCMAKE_INSTALL_LIBEXECDIR=/usr/bin"
+hostmakedepends="pkg-config qt6-tools qt6-base"
+makedepends="qt6-multimedia-devel taglib-devel qt6-svg-devel"
+depends="media-player-info avahi qt6-multimedia qt6-svg libmtp libcddb libmusicbrainz5 mpg123 taglib mpd udisks2"
```

Comment:
There is also no need for a hard dependency on mpd. Let the user decide if they want to use Cantata with a local MPD setup or with a remote one.